Remove script system

josealvim opened the pull request:
Closes issue #45

Commit:
Remove swig modules

	First commit in a series to solve issue 48
	which is: get rid of the script code

Commit:
Remove script from cmake file

Commit:
Finish deleting script files

	Now we need to fix our mad rampage

Commit:
Deal with the cmake files

	So that scripts are actually removed
	@this point we have not test compiled
	it though.

Commit:
Finilize (hopefully) Script removal from ugdk code

	Emphasis on hopefully, but cmake generated
	ok and make worked just fine sooooo, yeah!

Commit:
Change Travis to not do script tests

Commit:
Kills script remanents in cmake folder

	Because of course something had to go wrong.

Commit:
Running into problems

Commit:
Undo mistakes

	how could this happen to me
	I made a mistake
	.
	.
	.

Kazuo256 commented:
@henriquegemignani 
Seem like appveyor is breaking due to SDL2's cmake now. Also, I can't edit appveyors build setttings from my account =).

Commit:
Try to fix tests

Commit:
Fix tests

	Returned the glorious macro to it's rightful place in
	cmakelists.txt in the root directory
